db/migrate/20150729041232_add_piece_type_to_pieces.rb

```diff
@@ -0,0 +1,6 @@
+class AddPieceTypeToPieces < ActiveRecord::Migration
+
+  def change
+  	rename_column :pieces, :type, :piece_type
```

Comment:
strange, did it look like...

```Ruby
add_column :pieces, :type, :string
```

(i missed the error in slack, i'm so behind on slack :smiley:)